photon-lib/src/main/java/org/photonvision/PhotonCamera.java

```diff
@@ -111,20 +105,14 @@ public void close() {
 
     private long prevHeartbeatValue = -1;
     private double prevHeartbeatChangeTime = 0;
-    private static final double HEARBEAT_DEBOUNCE_SEC = 0.5;
+    private static final double HEARTBEAT_DEBOUNCE_SEC = 0.5;
 
     public static void setVersionCheckEnabled(boolean enabled) {
         VERSION_CHECK_ENABLED = enabled;
     }
 
     Packet packet = new Packet(1);
 
-    private static AprilTagFieldLayout lastSetTagLayout =
-            AprilTagFields.kDefaultField.loadAprilTagLayoutField();
-
-    // Existing is enough to make this multisubscriber do its thing
```

Comment:
Why are we removing this

Reply:
Currently unused and I will add back all this logic when we do protobuf ATFL

Reply:
Like I said in my comment, this field is _not_ unused.

Reply:
Oh sorry, I thought you were refering to the ATFL field, yes I accidentally removed the multisubscriber